_posts/2019-08-26-how-to-use-gruntwork-infrastructure-as-code-library.adoc

```diff
@@ -640,140 +640,124 @@ next two sections of the guide will walk you through how to use each of these.
 === Using Terraform Modules
 
 This section will show you how to use Terraform modules from the Gruntwork Infrastructure as Code Library. As an illustrative example,
-we'll deploy the `vpc-app`  Terraform module from https://github.com/gruntwork-io/module-vpc[module-vpc].
+we'll deploy the `vpc-app` Terraform module from https://github.com/gruntwork-io/terraform-aws-vpc[terraform-aws-vpc], using the https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/modules/networking/vpc[`vpc` service
+module] from the https://github.com/gruntwork-io/terraform-aws-service-catalog[`terraform-aws-service-catalog` repo].
 
 [.exceptional]
-IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `module-vpc`.
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `terraform-aws-vpc` and `terraform-aws-service-catalog`.
 
-You can use this module to deploy a production-grade VPC on AWS. For full background information on VPCs, check
+You can use these modules to deploy a production-grade VPC on AWS. For full background information on VPCs, check
 out link:/guides/networking/how-to-deploy-production-grade-vpc-aws[How to deploy a production-grade VPC on AWS].
 
-==== Create a wrapper module
+==== Get to know the Service Catalog module
 
 The Terraform modules in the Gruntwork Infrastructure as Code Library are intentionally designed to be unopinionated, so they do not
 configure `provider` or `backend` settings. Moreover, you will often use multiple modules from the Infrastructure as Code Library,
 rather than just one at a time. Therefore, the canonical way to consume a Terraform module from the Gruntwork
-Infrastructure as Code Library is to create a _wrapper module_ in one of your own Git repos.
+Infrastructure as Code Library is to use the _Service Catalog_ module from the Gruntwork `terraform-aws-service-catalog` repo.
 
-Let's assume you have a repo called `infrastructure-modules` and create a `vpc-app` wrapper module in it:
+Take a look at the `vpc-app` wrapper module (_service module_) to get familiar with how this module uses the underlying `vpc-app`
+module from `terraform-aws-vpc`.
 
 ----
-infrastructure-modules
-  └ networking
-    └ vpc-app
-      └ main.tf
-      └ outputs.tf
-      └ variables.tf
+terraform-aws-service-catalog
+  └ modules
+    └ networking
+      └ vpc
+        └ README.md
+        └ main.tf
+        └ outputs.tf
+        └ variables.tf
 ----
 
-==== Configure your providers
+==== Terraform and providers are configured
 
-Inside of `main.tf`, configure whatever Terraform providers you're using. Since the `vpc-app` module you're using in
-this guide is an AWS module, you'll need to configure the AWS provider:
+The service catalog module's `main.tf` file configures the Terraform providers to use. Since the `vpc-app` module in
+this guide is an AWS module, it configures the AWS provider in the configuration of Terraform itself in `main.tf`:
```

Comment:
This is actually not true. It only specifies required provider versions, but it does not actually configure an `aws` provider. If you follow this as is, you will actually run into a prompt asking to specify the AWS Region to deploy into.

The old docs are still accurate even for the service catalog flavor.